### ISSUE TITLE
RStudio does not run code

### ISSUE THREAD

**Issue by Sergio-ote** [rstudio/rstudio#12500]
### System details

    RStudio Edition : Desktop
    RStudio Version :  2022.12.0 Build 353
    OS Version      :  Windows 10 (updated)
    R Version       :  4.2.22

### Describe the problem in detail

Open a file located in a remote directory, for example, using a VPN. Close your VPN connection, close RStudio. Open RStudio, the file in the remote location should be opened as some sort of cache. RStudio will not run code (most of the times, for some reason this might work). When closing the files in the remote location, the problem is solved.

This may happen if the VPN connection expires when hibernating your laptop for some time. Uninstalling and updating R and RStudio did not solve the problem, only closing the files from the remote location did.

- [ ] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- `[x]` I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [ ] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- `[x]` I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by ronblum** [comment]
@Sergio-ote Thank you for raising this! We've run into some issues with remote drives, and I'm tagging this as the culprit. When the code doesn't run, what gets displayed or what else can you do? Is it only the code in the specific file that's affected, while other files and the console work?

**Comment by Sergio-ote** [comment]
@ronblum Hi!
Basically the IDE is blocked in general, when you press `Esc` or try to run a line of code, the console shows another line, but without the `>` at the beggining. When you try to close the programme, it gets stucked, and only closes when you hit the `X` symbol on the corner of the window a second time. It also gets stucked when you try to restart the session, and can stay like that indefinitely.

In summary, you cannot interact with any of the features of RStudio, only close it.

I have to say the problem arose suddenly, before I updated RStudio to the version I have in my issue, but RStudio was working fine before that and all of this was not raising any problems, in case it helps you narrow down the issue.

Let me know if you would need any other information,
Sergio
